core/domain/user_jobs_continuous.py

```diff
@@ -73,33 +73,32 @@ def _get_batch_job_manager_class(cls):
     # Public query methods.
     @classmethod
     def get_recent_notifications(cls, user_id):
-        """Gets a list of recent notifications to show on the user's
-        dashboard.
+        """Gets a list of recent notifications to show on the user's dashboard.
 
         Args:
-            user_id: str. The unique ID of the user.
+            user_id: str. The unique id of the user.
 
         Returns:
-            tuple(float, list(dict)). A 2-tuple with following entries:
-            - float. The time, in milliseconds since the Epoch, when the job was
-                queued.
-            - list(dict). A list of recent updates to explorations and feedback
-                threads; Each entry in the list is a dict with keys:
-                - 'type': str. Either feconf.UPDATE_TYPE_EXPLORATION_COMMIT or
-                    feconf.UPDATE_TYPE_FEEDBACK_MESSAGE.
-                - 'activity_id': str. The ID of the exploration being committed
-                    to or to which the feedback thread belongs.
-                - 'activity_title': str. The title of the activity.
-                - 'last_updated_ms': float. The time when the update was made,
-                    in milliseconds since the Epoch.
-                - 'author_id': str. The ID of the author who made the update.
-                - 'subject': str. A brief description of the notification.
+            tuple(job_queued_msec, recent_notifications), where:
+                job_queued_msec: float or None. The time when the job was
+                    queued in milliseconds since the epoch, or None if the job
+                    does not exist.
+                recent_notifications: list(dict). Each dict has the keys:
```

Comment:
Should be a list of domain objects.

Reply:
Shouldn't we also change the [model](https://github.com/oppia/oppia/blob/022c1b9a1fad92163786c3778417a3eb393484f8/core/storage/user/gae_models.py#L854), then, to reflect the fields it actually requires? It feels like this should be a larger job (maybe one we make an issue for) that does migrations/unit testing/etc, and outside of the scope of some tidy-up.